app/views/state_file/questions/ny_county/edit.html.erb

```diff
@@ -14,4 +20,37 @@
     <% end %>
     <%= f.continue %>
   <% end %>
+
+  <script type="text/javascript">
+    window.addEventListener("load", () => {
+      const nycCounties = [
+          "Bronx",
+          "Kings (Brooklyn)",
+          "Manhattan (see New York)",
```

Comment:
I know this is verbatim from the ticket but, I think we can just use Manhattan (New York) since it is the Manhattan Borough, New York County.

Reply:
The problem here is that the list of counties is taken from here:
https://github.com/codeforamerica/vita-min/blob/main/app/lib/efile/ny/school_districts.csv

I'm not sure if we should update this list. Maybe product management can give more info.